### PR TITLE
Implemented old functionality & added sorting to summary tables

### PR DESCRIPTION
**Pull request by ASztolc**

### CODE REVIEW

File: templates/report.html [hunk 1]
```diff
@@ -25,25 +25,29 @@ <h2>Environment</h2>
 
     <h1>Results</h1>
     <p>{{ n_tests }} tests ran in {{ duration }} seconds. </p>
-
+    
     {% for child in children recursive %}
-        <h2>{{ child.prefix }} {{ child.name }}</h2>
+        <h2 id="heading-{{ child.id }}{{ child.children[0].random_seed }}">{{ child.prefix }} {{ child.short_name }}</h2>
```
Comment (khornlund) [inline templates/report.html]: Can you remove the reference to `random_seed` and use a generic property?

File: templates/report.html [hunk 2]
```diff
@@ -25,25 +25,29 @@ <h2>Environment</h2>
...
+                        <th class="sortable numeric">Total xPassed Tests</th>
+                        <th class="sortable numeric">Total xFailed Tests</th>
+                    </tr>
+                </thead>
                 {% for inner_child in child.children -%}
-                    <tr class="results-table-row">
-                        <td onclick="scroll_to(this)" class="scroller" id="{{ inner_child.name }}">{{ inner_child.name }}</td>
+                <tbody class="results-table-row">
+                    <tr>
+                        <td onclick="scroll_to(this)" class="scroller" id="{{ inner_child.id }}{{ inner_child.children[0].random_seed }}">{{ inner_child.short_name }}</td>
```
Comment (khornlund) [inline templates/report.html]: Can you remove the reference to `random_seed` and use a generic property?

File: how_to_anaconda.txt [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# How to use anaconda
```
Comment (khornlund) [inline how_to_anaconda.txt]: Delete this file from version control (keep it locally if you want)

**Comment by khornlund** [comment]
Just a couple comments about a specific reference to `random_seed` - can we build this so it will work with any metadata attribute (eg. split the tests up by an attribute other than random_seed) ?

**Comment by ASztolc** [comment]
Removed references to random_seed, didn't need an additional metadata

**Pull request merged**